src/main/resources/processes/ums_simple_happy_path.bpmn20.xml

```diff
@@ -0,0 +1,249 @@
+          <![CDATA[false]]>
+        </modeler:initiator-can-complete>
+      </extensionElements>
+    </userTask>
+
+    <endEvent id="sid-A80ACAC5-5D98-43E1-9733-2BF6892A4D4D" name="Task complete"></endEvent>
+
+    <sequenceFlow id="sid-00A4E781-94DB-43CA-897C-B733F375D4FA"
+      sourceRef="requesterAssignTask" targetRef="assigneeReceiveTask">
+    </sequenceFlow>
+    <sequenceFlow id="sid-A4E49B6E-CDC7-4053-B74A-07F24A1D6527"
+      sourceRef="assigneeReceiveTask" targetRef="assignSetStatusToIP">
+    </sequenceFlow>
+    <exclusiveGateway id="status">
+    </exclusiveGateway>
+    <exclusiveGateway id="sid-7B7EFDD6-C5A8-48BA-BF1B-B9CFF9767ACC">
+    </exclusiveGateway>
+    <sequenceFlow id="sid-B6B845C6-1005-4342-8E08-7F4F7E87C693" sourceRef="sid-7B7EFDD6-C5A8-48BA-BF1B-B9CFF9767ACC" targetRef="sid-5111DC7D-256D-4440-92DC-0E21B8FD847E">
+      <conditionExpression xsi:type="tFormalExpression"><![CDATA[${status=="in progress"}]]></conditionExpression>
+    </sequenceFlow>
+    <exclusiveGateway id="sid-0FBBE7CD-4826-45DA-8E1C-D52A0EA1C843">
+    </exclusiveGateway>
+
+    <sequenceFlow id="sid-4B3F38AE-99D1-44CB-8DAD-90FE3C3E45EE"
+      sourceRef="assignSetStatusToTR" targetRef="sid-0FBBE7CD-4826-45DA-8E1C-D52A0EA1C843">
+    </sequenceFlow>
+
+    <serviceTask id="sid-5AC85C6E-EC36-4F83-87AF-D9EA43816586" name="Update Task as Complete"
+      flowable:delegateExpression="${updateTaskStatus}">
+    </serviceTask>
+
+    <sequenceFlow id="sid-E041A880-A672-4457-B91D-E5942DCF4DC6"
+      sourceRef="sid-5AC85C6E-EC36-4F83-87AF-D9EA43816586" targetRef="sid-A80ACAC5-5D98-43E1-9733-2BF6892A4D4D">
+    </sequenceFlow>
+    <sequenceFlow id="sid-08AC6BA7-D5B9-40B6-BC47-507A58E8ACE0"
+      sourceRef="requesterSetStatus" targetRef="status"
+    ></sequenceFlow>
+    <sequenceFlow id="sid-DC04B80F-AF6C-4077-B001-DABD90EFCD57"
+      sourceRef="status" targetRef="assignSetStatusToTR">
+      <conditionExpression xsi:type="tFormalExpression"><![CDATA[${status!="complete"}]]></conditionExpression>
+    </sequenceFlow>
+
+    <sequenceFlow id="sid-2739F20F-3522-4C45-BE7A-0E6281BB5FF6"
+      sourceRef="status" targetRef="sid-5AC85C6E-EC36-4F83-87AF-D9EA43816586">
+      <conditionExpression xsi:type="tFormalExpression"><![CDATA[${status=="complete"}]]></conditionExpression>
+    </sequenceFlow>
+
+    <serviceTask id="sid-3F3797F0-1D05-46C2-8CD3-6F9C2762D8DD" name="Update Task To Review"
+      flowable:delegateExpression="${updateTaskStatus}">
+    </serviceTask>
+
+    <sequenceFlow id="sid-27934CAE-B7AC-4673-9978-A797E1DB4B3F"
+      sourceRef="sid-3F3797F0-1D05-46C2-8CD3-6F9C2762D8DD" targetRef="requesterSetStatus">
+    </sequenceFlow>
+    <sequenceFlow id="sid-8D92F604-78C7-42B5-B5D0-C01F2AF4D24F"
+      sourceRef="sid-0FBBE7CD-4826-45DA-8E1C-D52A0EA1C843" targetRef="assignSetStatusToTR">
+      <conditionExpression xsi:type="tFormalExpression">
+        <![CDATA[${status!="to review"}]]>
```

Comment:
I think we should align status messages between the workflow and the API layer see enums: https://github.com/Maxar-Corp/UMS-flowable-spring-boot/pull/3/files#diff-aa46b0712a6b4ab96b3fcb3a1afb2721ec5c8470467d4979733b598084587b50

They currently are:     
PENDING("Pending"),
    TO_REVIEW("To Review"),
    IN_PROGRESS("In Progress"),
    COMPLETE("Complete");
    
    I'm open to whatever convention we want to go with...but I expect the request body will contain the enum string value

Reply:
Talked about making this change after discussions on merging capabilities.